components/CalendarCard/CalendarCard.tsx

```diff
@@ -0,0 +1,66 @@
+import { Skeleton } from 'components/Skeleton'
+import { IconCalendarSolid } from 'components/icons/components/IconCalendarSolid'
+import { IconLocationDotSolid } from 'components/icons/components/IconLocationDotSolid'
+import { IconEllipsisHSolid } from 'components/icons/components/IconEllipsisHSolid'
+
+interface UserProps {
+  name: string
+  avatar: string
+  date: string
+  time: string
+  location: string
+  isFirstLoading?: boolean
```

Comment:
The skeleton UI should be in a separate component. However, it is out of the scope of the task so I would say we should get rid of the part.